### PR TITLE
Paged efolder documents

### PR DESCRIPTION
**Pull request by pkarman**
connects #239

### CODE REVIEW

**Comment by leikkisa** [comment]
@pkarman Just checking if this has been tested against VBMS UAT.

**Comment by pkarman** [comment]
@leikkisa I have tested against VBMS production (that was the use case I had).

File: lib/vbms/requests/find_paged_document_series_references.rb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+              construct_response(version)
+            end
+          end
+          { paging: paging, documents: documents.flatten }
+        end
+      end
+
+      private
+
+      def construct_response(result)
```
Comment (leikkisa) [inline lib/vbms/requests/find_paged_document_series_references.rb]: should the argument here be called version?

File: spec/requests/find_paged_document_series_references_spec.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    it "generates valid SOAP" do
+      xml = subject.soap_doc.to_xml
+      xsd = Nokogiri::XML::Schema(fixture("soap.xsd"))
+      expect(xsd.errors).to eq []
+      errors = xsd.validate(parse_strict(xml))
+      expect(errors).to eq []
+    end
+  end
+
+  describe "parsing the XML response" do
```
Comment (leikkisa) [inline spec/requests/find_paged_document_series_references_spec.rb]: Nice tests!

**Pull request merged**